### PR TITLE
add wskadmin to ow-utils docker image

### PR DESCRIPTION
**Pull request by sciabarracom**
## Description

Adds a docker image containing wskadmin  to be used in kubernetes deployment for administration.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [x] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [x] I updated the documentation where necessary.

### CODE REVIEW

File: tools/admin/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM python:3
+COPY wskadmin /bin/wskadmin
+COPY wskprop.py /bin/wskprop.py
+COPY wskutil.py /bin/wskutil.py
+WORKDIR /bin
+CMD wskadmin
```
[the remark this answers is not in the document]
Comment (sciabarracom) [reply tools/admin/Dockerfile]: I documented the usage in the README for kube-deploy in a pending PR

**Comment by csantanapr** [comment]
We already have a multi purpose image wskutil we can add these there instead of new image

**Comment by csantanapr** [comment]
Here is the Dockerfile https://github.com/apache/incubator-openwhisk/blob/master/tools/ow-utils/Dockerfile

Let’s have one image for all these utilities wsk wskadmin etc...

**Comment by sciabarracom** [comment]
Ok I put the wskadmin there

File: tools/admin/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (dgrove-oss) [reply tools/admin/Dockerfile]: @rabbah -- I agree.  Let's put wsk admin into the ow-utils image

**Comment by sciabarracom** [comment]
Here you go once the image is published I can update this PR that requires it
https://github.com/apache/incubator-openwhisk-deploy-kube/pull/420

**Pull request merged**